content/applications/sales/sales/products_prices/ewallets_giftcards.rst

```diff
@@ -0,0 +1,143 @@
+value`. Finally, set the :guilabel:`Valid Until` period if applicable.
+
+Generated eWallets can be accessed through the :guilabel:`eWallets` smart button in the upper-right
+corner. From there, :guilabel:`Send` or :guilabel:`Share` the eWallets via email or a URL link.
+
+.. image:: ewallets_giftcards/ewallet-share.png
+   :align: center
+   :alt: eWallets send and share buttons
+
+Click on an eWallet to change the :guilabel:`Expiration Date`, :guilabel:`Partner`, or
+:guilabel:`Balance`. The :guilabel:`Code` of an eWallet *cannot* be changed, deleted, or duplicated.
+
+Gift cards
+==========
+
+.. _ewallet_gift/gift-cards:
+
+Gift cards can be purchased by customers, and in turn used as a payment method upon checkout at an
+eCommerce shop or :abbr:`PoS (Point of Sale)`.
+
+Before creating a new gift card program, it is necessary to first create gift cards as products. To
+do so, go to :menuselection:`Sales app --> Products --> Products` and :guilabel:`Create` a product.
+On the product template, configure the options as follows:
+
+- :guilabel:`Product Name`: enter a name for the gift card product
+- :guilabel:`Can be Sold`: enabled
+- :guilabel:`Product Type`: select :guilabel:`Service`
+- :guilabel:`Invoicing Policy`: select :guilabel:`Prepaid/Fixed Price`
+- :guilabel:`Create on Order`: select :guilabel:`Nothing`
+- :guilabel:`Sales Price`: enter the amount of the gift card
+
+.. note::
+  In order to have gift cards of different amounts, create multiple gift card products and modify
+  the :guilabel:`Sales Price` accordingly.
+
+Once the gift card product is created, go to :menuselection:`Sales app --> Products --> Gift cards
+& eWallet` to :guilabel:`Create` a gift card program. The following configuration options are
+available:
+
+- :guilabel:`Program Name`: enter a name for the gift card program
+- :guilabel:`Program Type`: select :guilabel:`Gift Card`
+- :guilabel:`Gift Card Products`: select the gift card product created earlier. Repeat the process
+  if you created gift card products of different amounts.
+- :guilabel:`Email template`: select the default :guilabel:`Gift Card: Gift Card Information`
+  template, or create a new template by clicking on the field, selecting :guilabel:`Search More`,
+  and then clicking :guilabel:`Create`.
+- :guilabel:`Print Report`: select :guilabel:`Gift Card`
+- :guilabel:`Currency`: select the currency to use for the gift card program
+- :guilabel:`Company`: select the company for which the program is valid and available
+- :guilabel:`Available On`: select the applications on which the program is valid and available
+- :guilabel:`Website`: select the website on which the program is valid and available. Leave this
+  field empty to include all websites.
+- :guilabel:`Point of Sale`: select the :abbr:`PoS (Point of Sale)` in which the program is valid
+  and available. Leave this field empty to include all :abbr:`PoS (Point of Sale)`.
+
+.. image:: ewallets_giftcards/giftcard-configuration.png
```

Comment:
[Compressing](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#images) this image reduces file size by ~3x.